test/parallel/test-process-load-env-file.js

```diff
@@ -87,3 +87,55 @@ describe('process.loadEnvFile()', () => {
     assert.strictEqual(child.code, 0);
   });
 });
+
+describe('process.loadEnvFile(path, { override: true })', () => {
+
+  it('should not override the original value', async () => {
+    process.env.BASIC = 'Original value';
+    const code = `
+      process.loadEnvFile(${JSON.stringify(validEnvFilePath)});
+      const assert = require('assert');
+      assert.strictEqual(process.env.BASIC, 'Original value');
+    `.trim();
+    const child = await common.spawnPromisified(
+      process.execPath,
+      [ '--eval', code ],
+      { cwd: __dirname },
+    );
+    assert.strictEqual(child.stderr, '');
+    assert.strictEqual(child.code, 0);
+  });
+
+  it('should override the original value', async () => {
+    process.env.BASIC = 'Original value';
+    const code = `
+      process.loadEnvFile(${JSON.stringify(validEnvFilePath)}, {override: true});
+      const assert = require('assert');
+      assert.strictEqual(process.env.BASIC, 'basic');
+    `.trim();
+    const child = await common.spawnPromisified(
+      process.execPath,
+      [ '--eval', code ],
+      { cwd: __dirname },
+    );
+    assert.strictEqual(child.stderr, '');
+    assert.strictEqual(child.code, 0);
+  });
+
+  it('supports passing options only', async () => {
+    process.env.BASIC = 'Original value';
```

Comment:
This doesn't pass env variables to spawned processes. Spawn function 3rd argument takes env key.